css/style.css

```diff
@@ -559,11 +559,14 @@ ul{
 }
 .quike-links ul li{
     margin-bottom: 25px;
+    /* padding-righ: 24px; */
+    margin-left: 120px;
```

Comment:
this creates overlap in the mobile view
<img width="160" alt="image" src="https://github.com/Iamdivyak/portfolio/assets/102896170/ab255284-440e-430e-8fbe-71a4b3559458">

try another alternative.
thankyou

Reply:
Okay, I didn't checked in Mobile view, Let me fix this

Reply:
delete the comments also